tests/sles4sap/saptune.pm

```diff
@@ -48,6 +48,9 @@ sub run {
     die "Command 'saptune daemon start' didn't start tuned"
       unless (tuned_is 'running');
 
+    # Skip test if saptune version is 1
+    return if (script_output("saptune version") =~ m/current active saptune version is '1'/);
```

Comment:
This is too generic. Please add an `and is_upgrade()` to the conditional so this is only skipped on migration/upgrade tests. After all, we want this test to fail if by any chance a saptune v1 is packaged in newer products.

Reply:
Added this, but the `saptune version` command only exists on saptune v2.  It's extremely  unlikely that the RPM will ship in v1 compatibility mode.